api/apps/api/src/modules/blm/values/repositories/typeorm-project-blm-repository.ts

```diff
@@ -24,6 +33,15 @@ export class TypeormProjectBlmRepository extends ProjectBlmRepository {
     projectId: string,
     defaults: ProjectBlm['defaults'],
   ): Promise<Either<CreateFailure, true>> {
+    if (await this.repository.findOne(projectId)) return left(alreadyCreated);
+
+    const projectBlm = await this.repository.create();
+    projectBlm.id = projectId;
+    projectBlm.defaults = defaults;
+    projectBlm.values = [];
+    projectBlm.range = [0, 0];
+    await this.repository.insert(projectBlm);
```

Comment:
I am afraid it may not work - AFAIR `id` would be still overridden by TypeORM. (`.query` would work tho)